jupyter-notebooks/analytics/user-guide/03_change_detection.ipynb

```diff
@@ -527,6 +528,16 @@
     "    # Select only the items with selected observed date\n",
     "    month_items = items_gdf.loc[items_gdf['observed'] == observed]\n",
     "    \n",
+    "    # The heatmap rendering requires 2 or more distinct geometries. If there is only one,\n",
```

Comment:
you have put this in the `render_items_for_month` for the dynamic map, but it seems like you might have intended to put it in the heatmap `render_items_for_month` below
https://github.com/planetlabs/notebooks/pull/155/files#diff-c99883be981354d381a1b15066d14eb3R598

Reply:
I've moved this to the correct section.